Nontrivial characters fillin, icon and kbd

trshemanske opened the pull request:
First edits on fillin, icon, and kbd

siefkenj commented on packages/docs/pages/reference/elements/fillin.mdx:
```diff
@@ -8,12 +8,48 @@ import {
 <SyntaxDisplay name="fillin" />
 
+### Remark
+`<fillin>` has publication options which govern the style 
```
`<fillin/>`

siefkenj commented on packages/docs/pages/reference/elements/fillin.mdx:
````diff
@@ -8,12 +8,48 @@ import {
+
+```ptx-example
+<FRAGMENT parents="p">
+    <c>&lt;m&gt;x^a * x^b = x^&lt;fillin fill="a+b" /&gt;&lt;/m&gt;</c> produces
````
I think it's more confusing than helpful to have the `<c>` in here. The extra `&lt;` entities make it harder to understand the code. Plus, they see the code and its rendered form already, so there is no need for "meta" examples.

siefkenj commented on packages/docs/pages/reference/elements/icon.mdx:
```diff
@@ -8,12 +8,17 @@ import {
 # `<icon>`
 
-Short description of what the `<icon>` tag does.
+The self-closing `<icon>` tag produces icons known to Font Awesome and which have a unicode representation.
```
`<icon/>`

trshemanske commented:
For clarification,

In giving the tag description, I was never sure whether it should be
rendered as `<tag>` or as `<tag/>`.  Many of the stubs have the former: Short
description of what the `<location>` tag does.
What is the convention?

I had forgotten that when finally rendered you could see both the source
and rendered code, so I shall delete the <c>...</c> lines.  Much easier for
me!

Tom


On Tue, Aug 13, 2024 at 11:30 AM Jason Siefken ***@***.***>
wrote:

> ***@***.**** commented on this pull request.
> ------------------------------
>
> In packages/docs/pages/reference/elements/fillin.mdx
> <https://github.com/siefkenj/pretext-book/pull/82#discussion_r1715506967>:
>
> >
>  ## Syntax
>
>  <SyntaxDisplay name="fillin" />
>
> +### Remark
> +`<fillin>` has publication options which govern the style
>
> <fillin/>
> ------------------------------
>
> In packages/docs/pages/reference/elements/fillin.mdx
> <https://github.com/siefkenj/pretext-book/pull/82#discussion_r1715509296>:
>
> > \ No newline at end of file
> +### A `<fillin/>` in math
> +
> +```ptx-example
> +<FRAGMENT parents="p">
> +    <c>&lt;m&gt;x^a * x^b = x^&lt;fillin fill="a+b" /&gt;&lt;/m&gt;</c> produces
>
> I think it's more confusing than helpful to have the <c> in here. The
> extra &lt; entities make it harder to understand the code. Plus, they see
> the code and its rendered form already, so there is no need for "meta"
> examples.
> ------------------------------
>
> In packages/docs/pages/reference/elements/icon.mdx
> <https://github.com/siefkenj/pretext-book/pull/82#discussion_r1715509624>:
>
> > @@ -8,12 +8,17 @@ import {
>
>  # `<icon>`
>
> -Short description of what the `<icon>` tag does.
> +The self-closing `<icon>` tag produces icons known to Font Awesome and which have a unicode representation.
>
> <icon/>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/siefkenj/pretext-book/pull/82#pullrequestreview-2235900400>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANHA5DHTF3M36EHZOFIRDRLZRIRDBAVCNFSM6AAAAABMOK4VD6VHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDEMZVHEYDANBQGA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Thomas R. Shemanske
Grantham, NH

siefkenj commented:
The `Short description of ...` is auto-generated and should be fully replaced.

The convention is, self closing tags are always referenced as self-closing. Tags that allow children are referenced by their opening tag.

trshemanske commented:
I think this addresses your comments.

Note that "kbd" tag has two variations, one self-closing and one not.  I detail this in the examples, but not sure if the body of this page should note the variants as I saw in another page.
